rasterio/enums.py

```diff
@@ -162,7 +162,11 @@ class OverviewResampling(IntEnum):
 
 
 class Compression(Enum):
-    """Available compression algorithms."""
+    """Available compression algorithms for GeoTIFFs.
+
+    Note that compression options for EXR, MRF, etc are not included
+    in this enum.
+    """
```

Comment:
In hindsight, I think a compression enum was a mistake. GDAL doesn't have an enum for this, and while `COMPRESS` is a common creation option, different drivers look at it in different ways. MRF conflates actual compression with image types, for example.